mappings/net/minecraft/world/gen/ChunkRandom.mapping

```diff
@@ -2,32 +2,76 @@ CLASS net/minecraft/class_2919 net/minecraft/world/gen/ChunkRandom
 	FIELD field_13276 sampleCount I
 	METHOD <init> (J)V
 		ARG 1 seed
-	METHOD method_12659 setSeed (II)J
-		ARG 1 x
-		ARG 2 z
-	METHOD method_12660 consume (I)V
-		ARG 1 count
-	METHOD method_12661 setSeed (JII)J
+	METHOD method_12659 setTerrainSeed (II)J
+		COMMENT Seeds the randomizer to generate the surface terrain blocks(such as grass, sand, etc)
+		COMMENT and the bedrock patterns.
+		COMMENT
+		COMMENT Note that the terrain seed is NOT world seed dependant and only gets affected by
+		COMMENT chunk coordinates.
+		ARG 1 chunkX
+		ARG 2 chunkZ
+	METHOD method_12660 skip (I)V
+		COMMENT Skips the provided number of calls to the randomizer.
+		COMMENT
+		COMMENT Functionally, this method serves no purpose since the combination of multiple linear
```

Comment:
It's not that this method serves "no" purpose, it serves the purpose to skip a certain number of calls. It's just that it's inefficient.

Reply:
The resulting seed is still linearly dependent, so useless.

Reply:
Ok, I guess it's misleading since it does "scramble" it in a sense.

Reply:
"scramble"?

I guess if you think of it like rolling a dice... Does it do the skipping by calling the randomizer and ignoring the result?

Reply:
I have trouble understanding the purpose of this function. I believe it is used to avoid finding similarities in terrain between the end and overworld generators... but since the resulting seed is still linearly dependant on the other ones so it doesn't *do much*. 

> I guess if you think of it like rolling a dice... Does it do the skipping by calling the randomizer and ignoring the result?

Yes, basically...